modules/admin_manual/pages/enterprise/security/oidc/index.adoc

```diff
@@ -0,0 +1,126 @@
+-- {openid-connect-url}, "What is OpenID Connect?"
+
+== Flows within ownCloud 10 server
+
+== Setup (Installation and Configuration)
+
+* xref:configure-owncloud[Configure ownCloud]
+* xref:setup-within-the-openid-provider[Setup Within the OpenID Provider]
+* xref:setup-service-discovery[Setup Service Discovery]
+* xref:how-to-setup-an-idp-for-development-and-test-purposes[How to Setup an IdP for Development and Test Purposes]
+* xref:integration-with-different-idps[Integration with different IdPs]
+
+=== Configure ownCloud
+
+Please refer to xref:configuration/server/config_sample_php_parameters.adoc#oidc-configuration[the OIDC configuration example] in the sample configuration documentation for how to configure ownCloud to work with OIDC.
+
+=== Setup Within the OpenID Provider
+
+When registering ownCloud as OpenID Client use `https://cloud.example.net/index.php/apps/openidconnect/redirect` as the redirect URL.
+
+In case {openid-connect-frontchannel-logout-url}[OpenID Connect Front-Channel Logout 1.0] is supported please enter `https://cloud.example.net/index.php/apps/openidconnect/logout` as the logout URL within the client registration of the OpenID Provider.
+We require `frontchannel_logout_session_required` to be true.
+
+=== Setup Service Discovery
+
+To allow other clients to use OpenID Connect when talking to ownCloud, please set up a redirect on the webserver to point `.well-known/openid-configuration` to `/index.php/apps/openidconnect/config`.
+Here is an .htaccess example
+
+[source]
+----
+RewriteRule ^\.well-known/openid-configuration /index.php/apps/openidconnect/config [R=301,L]
+----
+
+TIP: Service discovery is not mandatory at the moment as no client, currently, supports it.
+
+=== How to Setup an Identity Provider (IdP) for Development and Test Purposes
+
+There are various Open Source IdPs out there. 
+The one with the most implemented features seems to be {panva-node-oidc-provider-url}[panva/node-oidc-provider].
+
+To set it up locally do the following:
+
+. Clone {panva-node-oidc-provider-url}[panva/node-oidc-provider]
+. Run `yarn install`
+. cd example
+. Add the client config into https://github.com/panva/node-oidc-provider/blob/master/example/support/configuration.js#L14
++
+[source,php]
+----
+module.exports.clients = [
+    {
+        client_id: 'ownCloud',
+        client_secret: 'ownCloud',
+        frontchannel_logout_uri: 'http://localhost:8080/index.php/apps/openidconnect/logout',
+        grant_types: ['refresh_token', 'authorization_code'],
+        redirect_uris: ['http://localhost:8080/index.php/apps/openidconnect/redirect'],
+    }
+];
+----
+. Start the IdP via: `node standalone.js`
+. Open {openid-config-url} in your browser.
+  Your ownCloud configuration should look as follows:
```

Comment:
Proposal:
`Your ownCloud configuration should look as follows:` —>
`Your ownCloud configuration key as part of your „config/config.php“ file should look as follows:`

Reply:
The following is a nice compromise: "_Your ownCloud configuration (`config/config.*.php`) should look as follows:_"